### PR TITLE
qf-fmt: make fmt

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

File: core/kazoo_endpoint/test/kz_formatters_test.erl [hunk 1]
```diff
@@ -32,28 +32,28 @@
 
 regex_inbound_test_() ->
     Formatter = kz_json:from_list([{<<"to">>
-                                   ,[kz_json:from_list([{<<"regex">>, <<"^\\+?1?(\\d{10})$">>}
+                                   ,[kz_json:from_list([{<<"regex">>, <<"^\\+?1?(\\d{10})\$">>}
```
Comment (k-anderson) [inline core/kazoo_endpoint/test/kz_formatters_test.erl]: @fenollp I know the CI checks failed due to the number manager test, but that means I can't tell if all the \$ changes are actually required or accidental...

Comment (fenollp) [inline core/kazoo_endpoint/test/kz_formatters_test.erl]: https://bugs.erlang.org/browse/ERL-204

File: core/amqp_leader/src/amqp_leader_proc.erl [hunk 2]
```diff
@@ -47,8 +47,8 @@
               ,node = node()           :: atom()
               ,name                    :: atom()
               ,sync                    :: any()
-%              ,candidates             :: atoms()
-             }).
+                                          %% ,candidates             :: atoms()
```
Comment (mark2600) [inline core/amqp_leader/src/amqp_leader_proc.erl]: delete this, don't leave dead code

File: core/kazoo/src/kz_network_utils.erl [hunk 3]
```diff
@@ -17,7 +17,7 @@
 -export([to_cidr/1
         ,to_cidr/2
         ,verify_cidr/2
-%%        ,expand_cidr/1
+         %% ,expand_cidr/1
```
Comment (mark2600) [inline core/kazoo/src/kz_network_utils.erl]: delete this also please

**Pull request merged**